src/main/java/org/spongepowered/api/data/Keys.java

```diff
@@ -2437,6 +2437,11 @@ public final class Keys {
      */
     public static final Key<Value<Ticks>> MIN_SPAWN_DELAY = Keys.key(ResourceKey.sponge("min_spawn_delay"), Ticks.class);
 
+    /**
+     * Location for the model of an {@link ItemStack}
```

Comment:
Can we link the wikidocs for what this model means? Since it's a texture model identifier that's available in a possible resource pack, it's a good idea to make it clear here.